src/test/java/org/entando/kubernetes/controller/mockmvc/InstallFlowTest.java

```diff
@@ -246,6 +246,9 @@ void shouldCallCoreToInstallComponentsWithInstallPlanRequest() throws JsonProces
         String stringInstallPlan = new ObjectMapper().writeValueAsString(TestInstallUtils.mockInstallWithPlansRequestWithActions());
         EntandoBundleJobEntity entandoBundleJobEntity = bundleJobEntityList.get(0);
         assertThat(entandoBundleJobEntity.getInstallPlan()).isEqualTo(stringInstallPlan);
+
+        // check that db custom_install column is correctly populated
+        assertThat(entandoBundleJobEntity.getCustomInstallation()).isTrue();
```

Comment:
@firegloves please use "is" for flags (instead of "get").
You are indeed doing it for the install plan